teams/teams-ps/teams/New-Team.md

```diff
@@ -12,8 +12,11 @@ ms.reviewer:
 
 ## SYNOPSIS
 This cmdlet lets you provision a new Team for use in Microsoft Teams and will create an O365 Unified Group to back the team. 
-Groups created through teams cmdlets, APIs, or clients will not show up in Outlook by default. 
-If you want these groups to appear in Outlook clients, you can use the [Set-UnifiedGroup](https://docs.microsoft.com/powershell/module/exchange/set-unifiedgroup) cmdlet in the Exchange Powershell Module to set the parameter HiddenFromExchangeClients to $false.
+Groups created through teams cmdlets, APIs, or clients will show up in Outlook by default. 
```

Comment:
```suggestion
Groups created through teams cmdlets, APIs, or clients will not show up in Outlook by default. 
```

Reply:
The default is that they do show up (i.e. if I provision a SharePoint Online Team site). Teams via the TAC (used to?) not show up.

Reply:
Having found time to test this today, `New-Team` is creating groups with both `HiddenFromAddressListsEnabled` and `HiddenFromExchangeClientsEnabled` set to true, and happy to report this applies to both module version 1.1.3 and 1.1.4. Odd that I was seeing different when I raised this but glad it's the default!